requirements/constraints.txt

```diff
@@ -12,8 +12,15 @@
 # This file contains all common constraints for edx-repos
 -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt
 
-# celery 5.0 has dropped python3.5 support.
-celery<5.0
+# As it is not clarified what exact breaking changes will be introduced as per
+# the next major release, ensure the installed version is within boundaries.
+celery>=5.2.2,<6.0.0
+
+# required for celery>=5.2.0;<5.3.0
+click>=8.0,<9.0
+
+# NOTE: Needed for Lilac + Celery>=5
```

Comment:
Please change it to TODO. You can copy-paste it from `setuptools` - that one is nice.

Reply:
Updated